stratum/hal/lib/common/config_monitoring_service.cc

```diff
@@ -326,7 +326,9 @@ ::grpc::Status ConfigMonitoringService::DoGet(::grpc::ServerContext* context,
       // `msg` PROTOBUF to the response that will be sent to the controller.
       InlineGnmiSubscribeStream stream(
           [resp](const ::gnmi::SubscribeResponse& msg) -> bool {
-            if (!msg.has_update()) return false;
+            //If update is empty, then it may be a sync_response for get request
```

Comment:
Add space before comment. Here is the [Style Guide](https://google.github.io/styleguide/cppguide.html#Comment_Style) Stratum follows.

Reply:
Done.